tests/unit/test_core.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+# #!/usr/bin/env python
```

Comment:
Just inline with other files. IMHO all those should be gone

On December 19, 2015 12:15:35 AM EST, kun <notifications@github.com> wrote:
>> @@ -1,4 +1,4 @@
>> -#!/usr/bin/env python
>> +# #!/usr/bin/env python
>
>2 `#`?
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/gabrielfalcao/HTTPretty/pull/279/files#r48089036